src/Tizen.Multimedia.MediaPlayer/Player/Player.Properties.cs

```diff
@@ -523,6 +523,31 @@ public bool IsAudioOnly
             }
         }
 
+        /// <summary>
+        /// Gets or sets the player's replaygain state.
+        /// </summary>
+        /// <value>If the replaygain status is true, replaygain is applied (if contents has a replaygain tag);
+        /// otherwise, the replaygain isn't affected by tag and properties.</value>
+        /// <exception cref="ObjectDisposedException">The player has already been disposed of.</exception>
+        /// <exception cref="InvalidOperationException">The player is not in the valid state.</exception>
+        /// <since_tizen> 5 </since_tizen>
+        public bool Replaygain
+        {
+            get
+            {
+                ValidateNotDisposed();
+                NativePlayer.IsReplaygain(Handle, out var value).
```

Comment:
Same above comment. IsReplaygain -> isReplayGain

Reply:
ok i'll fix it